raiden/tests/utils/transfer.py

```diff
@@ -76,6 +77,36 @@ def get_channelstate(
     return channel_state
 
 
+def transfer_and_check(
+    chain: List[App],
```

Comment:
also, please don't give more data to a function than it needs, I would expect the function to use these arguments, something like `transfer_and_assert_path`. Which is not true.